misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/core/BTestRunner.java

```diff
@@ -331,9 +336,9 @@ public void packageProcessed(BLangPackage bLangPackage, TestarinaClassLoader cla
         // TODO the below line is required since this method is currently getting explicitly called from BTestRunner
         TestSuite suite = TesterinaRegistry.getInstance().getTestSuites().get(bLangPackage.packageID.toString());
 
-        if (!bLangPackage.hasTestablePackage()) {
-            return;
-        }
+//        if (!bLangPackage.hasTestablePackage()) {
+//            return;
+//        }
```

Comment:
If the code is not required shall we remove ? 

What will be the behavior if you run `ballerina test moduleName` if the module do not have tests.

Reply:
I will remove this particular block out. 

When we run `ballerina test <moduleName>`, we get the following output on the command line
```
Compiling source
        mohamedaquibzulfikar/module1:0.1.0

Creating balos
        target/balo/module1-2019r3-any-0.1.0.balo

Running tests
        mohamedaquibzulfikar/module1:0.1.0
        No tests found
```

Reply:
Done